src/brod_auth_backend.erl

```diff
@@ -0,0 +1,3 @@
+-module(brod_auth_backend).
```

Comment:
It should be a part of brod, since it's brod to define the API and the backend application to implement it.